2-mandatory/2-financial-times.js

```diff
@@ -4,38 +4,79 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
+
 function potentialHeadlines(allArticleTitles) {
     // TODO
+    const headlines = [];
+    for (let articleTitle of allArticleTitles) {
+        if (articleTitle.length <= 65) {
+            headlines.push(articleTitle);
+        }
+    }
+    return headlines;
 }
 
+
+
+
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
+
+
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
This solution will return the title with the fewest characters, but this might be different from the title with the fewest words.
Can you try fixing this? (you only need to make 2 small changes 😄)